### PR TITLE
Eyewheel

### PR DESCRIPTION
**Pull request by H-M-H**
-fixed some stuff Chairn forgot
-I already tested but would be nice if someone else could have a look too

### CODE REVIEW

File: src/game/client/components/emoticon.cpp [hunk 1]
```diff
@@ -37,7 +40,9 @@ void CEmoticon::OnReset()
 {
 	m_WasActive = false;
 	m_Active = false;
+	//m_DrawEmotes = false;
```
Comment (nzyuzin) [inline src/game/client/components/emoticon.cpp]: Commented out?

File: src/game/client/components/emoticon.cpp [hunk 2]
```diff
@@ -134,6 +140,46 @@ void CEmoticon::OnRender()
 
 	Graphics()->QuadsEnd();
 
+	CServerInfo pServerInfo;
+	Client()->GetServerInfo(&pServerInfo);
+	if((IsDDRace(&pServerInfo) || IsDDNet(&pServerInfo) || IsPlus(&pServerInfo)) && g_Config.m_ClEyeWheel)
```
Comment (nzyuzin) [inline src/game/client/components/emoticon.cpp]: Maybe this check could be moved to a helper function?
Same check here as on line 209.

File: src/engine/shared/serverbrowser.cpp [hunk 3]
```diff
@@ -25,3 +25,8 @@ bool Is64Player(const CServerInfo *pInfo)
 	    || str_find(pInfo->m_aName, "64")
 	    || IsDDNet(pInfo);
 }
+
+bool IsPlus(const CServerInfo *pInfo)
+{
+	return str_find(pInfo->m_aGameType, "+");
+}
```
Comment (nzyuzin) [inline src/engine/shared/serverbrowser.cpp]: No newline at end of file.

**Comment by Chairn** [comment]
Happy that someone used my work ;).

**Comment by CytraL** [comment]
I only did it first, not contributed in this pr.... All thanks to Chairn and H-M-H ;)

**Comment by H-M-H** [comment]
okay, testest again and found no bugs so far, let's get it in.

**Pull request merged**